c7n/policy.py

```diff
@@ -207,6 +213,29 @@ def run(self, *args, **kw):
                 "ActionTime", time.time() - at, "Seconds", Scope="Policy")
             return resources
 
+    def get_logs(self, start, end):
+        log_source = self.policy.ctx.output
+        log_gen = ()
+        if log_source.use_s3():
+            session = utils.local_session(self.policy.session_factory)
+            raw_entries = log_entries_from_s3(session, log_source, start)
+            # log files can be downloaded out of order, so sort on timestamp
+            # log_gen isn't really a generator once we do this, but oh well
+            log_gen = sorted(
+                normalized_log_entries(raw_entries),
```

Comment:
part of the intent was that we could normalize on object keys rather than parse of entries